### DISCUSSION TITLE
🚀 Nota is great!

### DISCUSSION THREAD

**Discussion by minthemiddle** [notaapp/talk#341] [Random]
Hi,

just a year's end heads up and motivational message:
Nota is great! It's lean, it's fast, it does not have bells and whistles, it lets me focus on writing and working with my notes.
It does not distract me with lots of plugins and graphs and canvases (Obsidian, I'm looking at you!). Ah, and it looks very beautiful!

Nota is like Things, Obsidian like Ticktick. More features are not better for an user, a better philosophy and consistency and attention to detail are.

Great work!

**Comment by astoilkov** [comment]
That's what I call a Christmas present. Thank you for the words.

This is one of our core values. We are cautious with the "adding" part.

We also think from first principles and try to understand the basics. I think this will produce some great results in the next 1-3 years so stick around!

**Reply by astoilkov** [reply]
Also, new features (especially visual) get a lot more attention. Receiving appreciation for the opposite is very much appreciated.

Thank you!

**Comment by borekb** [comment]
Agreed, Nota is a beautiful piece of software! ❤️

**Comment by harveyr** [comment]
Agreed! Nota is a truly impressive blend of simplicity and power. I can't believe this is just the beta. And, I appreciate the calmer release cadence. I like to think it reflects the thoughtfulness of the dev team. Amazing work!

**Reply by erusev** [reply]
Thanks so much 😊 Feedback like this means a lot ❤️
